src/eth/primitives/call_input.rs

```diff
@@ -5,5 +5,6 @@ use crate::eth::primitives::Bytes;
 pub struct CallInput {
     pub from: Address,
     pub to: Option<Address>,
+    #[serde(default)]
```

Comment:
This field may not be present in the request payload, so we assume empty as default.